demos/simple-menu.html

```diff
@@ -35,7 +35,11 @@
         position: relative;
         flex: 1;
         margin-top: 16px;
-        background-color: antiquewhite;
```

Comment:
Let's leave this as is. The gray being used here has a bit too low a contrast IMHO.